### ISSUE TITLE
Error using "using namespace" in function

### ISSUE THREAD

**Issue by tenk-a** [LADSoft/OrangeC#44]
Using "using namespace" in a function may cause an error (and occ.exe crash)
 (v6.0.27.1)

smp.cpp:
```
#include <ratio>
#include <chrono>
int main() {
    using namespace std::chrono;
    //typedef duration<double, std::ratio<1> > t;    // error
    typedef std::chrono::duration<double, std::ratio<1> > t;   // error
}
```
cmdline> occ smp.cpp
RESULT:
Error(437)    C:\Program Files (x86)\Orange C 386\include\chrono(452):  Invalid use of type '' in function main
Warning(436)  smp.cpp(6):  In template instantiation started here
Error( 16)    C:\Program Files (x86)\Orange C 386\include\chrono(452):  Identifier expected in function main
Warning(436)  smp.cpp(6):  In template instantiation started here
Error( 11)    C:\Program Files (x86)\Orange C 386\include\chrono(452):  Syntax error: ; expected in function main
Warning(436)  smp.cpp(6):  In template instantiation started here
Error(414)    C:\Program Files (x86)\Orange C 386\include\chrono(459):  Cannot use template 'std::__1::chrono::duration<_Rep, _Period>' without specifying speci
alization parameters in function main
Error( 11)    C:\Program Files (x86)\Orange C 386\include\chrono(459):  Syntax error: ) expected in function main
(occ.exe crash)

Can be compiled by removing "using namespace std::chrono;"

**Comment by LADSoft** [comment]
fixed in version 6.0.28

**Comment by tenk-a** [comment]
I compiled with v 6.0.28.1.
Thank you.
